cms/templates/import.html

```diff
@@ -113,11 +165,23 @@ <h3 class="title">${_("Updating Course")}</h3>
 
               <div class="status-detail">
                 <h3 class="title">${_("Success")}</h3>
-                <p class="copy">${_("Your imported content has now been integrated into this course")}</p>
+                <p class="copy">
```

Comment:
I know this isn't in scope for the story, but if we do any additional updates to this PR, would it be possible to change line 163 in this file to use the correct icon? This is a bug introduced by an icon refactor a month back or so, and I just rediscovered it while testing. (Current: fa-square-o) --> Proposed fa-check